stable/database/templates/statefulset.yaml

```diff
@@ -495,8 +526,39 @@ spec:
           # the SM becomes unready if it does not start within 15 minutes = 30s + 15s*58
           successThreshold: 2
           timeoutSeconds: {{ default 5 .Values.database.sm.readinessTimeoutSeconds }}
+{{- if and .Values.database.insights .Values.insights.enabled }}
+      - name: insights
+        image: {{ template "insights.image" . }}
+        imagePullPolicy: {{ .Values.insights.image.pullPolicy }}
+        tty: true
+        volumeMounts:
+        - mountPath: /etc/telegraf/telegraf.d/
+          name: insights-config
+        - name: log-volume
+          mountPath: /var/log/nuodb
+      - name: insights-config
+        image: {{ template "insights.watcher" . }}
+        imagePullPolicy: {{ .Values.insights.watcher.pullPolicy }}
+        env:
+        - name: LABEL
+          value: {{ .Values.database.insights | quote }}
+        - name: FOLDER
+          value: /etc/telegraf/telegraf.d/
+        - name: REQ_URL
+          value: http://0.0.0.0:5000/reload
+        volumeMounts:
+        - name: insights-config
+          mountPath: /etc/telegraf/telegraf.d/
+        - name: log-volume
+          mountPath: /var/log/nuodb
+      shareProcessNamespace: true
+{{- end }}
 {{- include "nuodb.imagePullSecrets" . | indent 6 }}
       volumes:
+      {{- if and .Values.database.insights .Values.insights.enabled }}
+      - name: insights-config
+        emptyDir: {}
```

Comment:
I remember seeing some configmaps on the demo. How they are configured? Should they be part of the admin chart?
It looks like the `insights-config` is ephemeral storage...

Reply:
The configmaps are loaded by a different chart, current in nuodb-dashboard-insights.

Reply:
Do you plan to migrate the chart here? I think that we should mention the dependencies. Users might think that it's enough to set `insights` to `true` but it seems not the case.

Reply:
It's probably more of a naming issue than anything.    The architecture is one where a collector (nuocd) runs at the endpoints that you are monitoring.   It will push data to a server.   What that server is,  is up to user to configure along with the plugins to push to the server.  Out of the box, we ship with a server that configures plugins to collect and ship to influx.  The term insights is being used at both the collector and our default server.

Reply:
Thanks for the explanation! My takeaway is that another chart from `nuodb-dashboard-insights` is needed to get the "server" along with the collector configuration.

Reply:
yes,  until you install the chart insights in nuodb-dashboard-insights.  The sidecar is dormant.   Once you install the plugins they will be picked up by the sidecar and it will start collecting the metrics.  If there is an output plugin defined then those metrics are pushed to the output endpoint.